compiler/modules/types/error.bal

```diff
@@ -1,6 +1,6 @@
 // Implementation specific to basic type error.
 
-public function errorDetail(Context cx, SemType detail) returns SemType {
+public function errorDetail(SemType detail) returns SemType {
```

Comment:
Let's not do this in this PR.

Reply:
1) Currently type sexpr parser only refer to t:Env, never to t:Context.
2) `Context cx` was only added with Lochana's changes, even then it was never used.

Therefor, I propose we remove it in this PR.